app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,22 +1,26 @@
 import React, { useState, useEffect } from "react";
 
 import EmptyNotesListImage from "images/EmptyNotesList";
-import { Button, PageLoader } from "neetoui";
-import { Header, SubHeader } from "neetoui/layouts";
+import { PageLoader } from "neetoui";
 
 import notesApi from "apis/notes";
 import EmptyState from "components/Common/EmptyState";
+import { getInitialSidebarLink } from "helpers/notes";
 
 import DeleteAlert from "./DeleteAlert";
 import NewNotePane from "./NewNotePane";
-import NoteTable from "./NoteTable";
+import NotesCategories from "./NotesCategories";
+import NotesHeader from "./NotesHeader";
+import NotesList from "./NotesList";
 
 const Notes = () => {
   const [loading, setLoading] = useState(true);
   const [showNewNotePane, setShowNewNotePane] = useState(false);
   const [showDeleteAlert, setShowDeleteAlert] = useState(false);
-  const [searchTerm, setSearchTerm] = useState("");
-  const [selectedNoteIds, setSelectedNoteIds] = useState([]);
+  const [showCategoryPane, setShowCategoryPane] = useState(true);
```

Comment:
Let's always define boolean states with the following convention - 
`const [isCategoryPaneOpen, setIsCategoryPaneOpen] = useState(true);`